mysql-test/suite/tokudb.perfschema/t/bug-ps-4711.test

```diff
@@ -0,0 +1,18 @@
+# PS-4711 test
+# All PFS objects deinitialization must be before PFS shutdown, otherwise
+# there will be crash on server shutdown
+
+--source include/not_embedded.inc
+--source include/have_perfschema.inc
+--source include/have_tokudb.inc
+
+--echo # Stop DB server which was created by MTR default
+--exec echo "wait" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
+--shutdown_server
+--source include/wait_until_disconnected.inc
```

Comment:
why not --source include/shutdown_mysqld.inc and --source include/start_mysqld.inc ?

Reply:
--source include/restart_mysqld.inc must be used here

Reply:
fixed

Reply:
https://jenkins.percona.com/view/5.7/job/mysql-5.7-param/1900/